packages/locales/package.json

```diff
@@ -7,21 +7,9 @@
   "types": "./lib/index.d.ts",
   "license": "MIT",
   "scripts": {
-    "build": "wireit",
     "release": "npm publish",
     "release:dry": "npm publish --tag latest --dry-run"
   },
-  "wireit": {
-    "build": {
-      "command": "node ./build.js",
-      "files": [
-        "src/locales/*.json"
-      ],
-      "output": [
-        "lib/locales/*.js"
-      ]
-    }
-  },
```

Comment:
A custom build step was used to copy JSON locale files as JavaScript files that can be imported. This is no longer needed with JSON modules.